xml/Microsoft.Build.Framework/SdkReference.xml

```diff
@@ -79,10 +79,9 @@
         <Parameter Name="other" Type="Microsoft.Build.Framework.SdkReference" />
       </Parameters>
       <Docs>
-        <param name="other">An object to compare with this object.</param>
-        <summary>Indicates whether the current object is equal to another object of the same type.</summary>
-        <returns>
-          <see langword="true" /> if the current object is equal to the <paramref name="other" /> parameter; otherwise, <see langword="false" />.</returns>
+        <param name="other" />
```

Comment:
Losing some valuable comments here. Can you check if this is happening anywhere else?

Reply:
So far I've determined that MSBuild sources never had those comments, so this is one case where probably, the docs XML files had content, but MSBuild sources did not.